pkgs/by-name/mg/mgitstatus/package.nix

```diff
@@ -0,0 +1,33 @@
+{ fetchFromGitHub, lib, stdenv }:
+
+stdenv.mkDerivation rec {
+  pname = "mgitstatus";
+  version = "2.2";
+
+  src = fetchFromGitHub {
+    owner = "fboender";
+    repo = "multi-git-status";
+    rev = version;
+    hash = "sha256-jzoX7Efq9+1UdXQdhLRqBlhU3cBrk5AZblg9AYetItg=";
+  };
+
+  installPhase = ''
+    runHook preInstall
+
+    mkdir -p $out/bin $out/man/man1
+    install -m 755 mgitstatus $out/bin/${pname}
+    install mgitstatus.1 $out/man/man1
+
+    runHook postInstall
+  '';
+
+  meta = with lib; {
```

Comment:
suggestion: add `mainProgram`

Reply:
Oh, I thought this needed to be specified when a derivation provides multiple binaries. TIL otherwise.